### ISSUE TITLE
Request changing xcatprobe xcatmn output messages for checking services

### ISSUE THREAD

**Issue by whowutwut** [xcat2/xcat-core#2320]
@hu-weihua  From my last comment in Issue #2193 , I think we should improve this messages for checking services can be improved..

Looking at the current output: 

```
[root@fs4 ~]# xcatprobe xcatmn -i enP4p1s0f2
<mn>: All xCAT deamons are running                                                                                [ OK ]
<mn>: xcatd can receive command request                                                                           [ OK ]
<mn>: 'site' table is configured correctly                                                                        [ OK ]
<mn>: Provision network is configured correctly                                                                   [ OK ]
<mn>: 'passwd' table is configured correctly                                                                      [ OK ]
<mn>: Important directory is configured correctly                                                                 [ OK ]
<mn>: SELinux is disabled on current server                                                                       [ OK ]
<mn>: HTTP service works well                                                                                     [ OK ]
<mn>: TFTP service works well                                                                                     [ OK ]
<mn>: DNS service works well                                                                                      [FAIL]
<mn>: DNS service isn't ready on 192.168.3.25                                                                     
<mn>: DHCP service works well                                                                                     [ OK ]
<mn>: Firewall is closed on current server                                                                        [ OK ]
<mn>: The disk space is enough for xCAT to work                                                                   [ OK ]
<mn>: The IP of master is a static IP address                                                                     [ OK ]
<mn>: The dhcpd.leases file is less than 100M                                                                     [ OK ]
======================do summary=====================                                                             
[MN]: Check on MN FAILED.                                                                                         [FAIL]
    DNS service works well                                                                                        [FAIL]
        DNS service isn't ready on 192.168.3.25                                                                   
```

It seems like we are writing statements and then the response is "OK" or "FAIL". 

Looking at the full list, I have some problem with the following messages:

* Important directory is configured correctly  

  What is the important directory? If it's not configured correctly, what is the user to do? 

* \<SERVICE\> service works well 

   Suggest changing to \<SERVICE\> is configured... or Checking \<SERVICE\> is configured...

* The disk space is enough for xCAT to work

   Suggest printing out the testing threshold.   ==> "Checking for mininum availble disks space of XX GB..."

* The IP of master is a static IP address 

   Suggest:  The xCAT management node IP: \<IP ADDRESS\> is configured to static

* Firewall is closed on current server 

   Suggest: Firewall is disabled 

* SELinux is disabled on current server, suggest removing "current server"  because the probe is checking THIS server.   (We may need to potentially support xCAT with SELinux enabled... just FYI)

**Comment by hu-weihua** [comment]
@whowutwut ， thanks for your comments. I will refine the output of ``xcatmn`` depending on your comments.

**Comment by hu-weihua** [comment]
@whowutwut , I have refine the code depending on your comment by PR #2370. Could you help to review the change, thanks.

**Comment by hu-weihua** [comment]
@whowutwut , could you help to verify this issue? if there is not problem, could you close this issue? thanks.

**Comment by whowutwut** [comment]
Closing this out.  The output is improved.

I did have a few more very minor comments that I will just document here.  I won't open an issue for it.  

1)  Maybe delete "All" and just write "xCAT daemons are running"
```
[root@fs4 ~]# xcatprobe xcatmn -i enP4p1s0f2
<mn>: All xCAT deamons are running                                   
```

2)  Change the `<mn>` to be in sync with the summary spelling `<MN>`

3) The "do summary" should just be changed to "Summary", and maybe changing the output message to "Checking in Management Node Passed"

```
======================  Summary =====================
[MN]: Check on MN PASS.                                           
```

If I have some time when you are on holiday, or @gurevichmark if you have some time..  we can create a pull request for this.
